generative_ai/tune_code_generation_model.py

```diff
@@ -16,10 +16,15 @@
 from __future__ import annotations
 
 
+from typing import Optional
+
+
 from google.auth import default
+from google.cloud import aiplatform
 import pandas as pd
 import vertexai
-from vertexai.preview.language_models import CodeGenerationModel
+from vertexai.preview.language_models import CodeGenerationModel, TuningEvaluationSpec

```

Comment:
Similar to `from google.cloud import aiplatform`, import modules rather than individual classes. I can see that this pattern has been used throughout, so feel free to file an issue and follow up in a different PR instead.

Reply:
Will follow up in a different PR. As previously I had tried, but only import modules didn't work. It needed to specify the model name in import.

Reply:
```
>>> from vertexai.preview import language_models 
>>> language_models.CodeGenerationModel()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: _LanguageModel.__init__() missing 1 required positional argument: 'model_id'
```

Seems to work?